float32 translation 	#translation of spindle caddy in meters
float32 rotation	#rotation of arm in radians
float32 extention	#extension of finger in meters
